client/api.py

```diff
@@ -25,14 +25,14 @@ def health_check():
 
 @app.route('/website', methods=['GET'])
 def get_metadata():
-    query_params = QueryParams()
+    urls = request.args.getlist('src')
```

Comment:
just wondering does this actually work if there isn't a list of urls?
I think this would return a list even if there is only 1 url, which is not what we want.